### PR TITLE
feat(fa): Staff mock

### PR DESCRIPTION
**Pull request by Rikkiabb**
# ...
https://app.asana.com/0/1200897217304888/1200980104456794

## What
Hard coding staff mocks.
Creates staff controller, module, model, dto and service
Moved bunch of stuff around 🙈 

## Why
Initial setup for working with applicaitons

## Checklist:

- [x] I have performed a self-review of my own code
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] Formatting passes locally with my changes
- [x] I have rebased against main before asking for a review

### CODE REVIEW

File: apps/financial-aid/backend/src/app/modules/staff/staff.service.ts [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  constructor(
+    @InjectModel(StaffModel)
+    private readonly staffModel: typeof StaffModel,
+  ) {}
+
+  findById(id: string): Promise<Staff> {
+    const mockStaff: Staff = {
+      id: id,
+      name: 'Klára Línudóttir',
+      nationalId: '0000000002',
```
Comment (Rikkiabb) [inline apps/financial-aid/backend/src/app/modules/staff/staff.service.ts:19]: Not sure if we want to use the user obj here or hard code this, any thoughts?

File: libs/financial-aid/shared/src/lib/types.ts [hunk 2]
```diff
@@ -1,212 +1,5 @@
...
-export interface User {
-  nationalId: string
-  name: string
-  phoneNumber: string
-  folder: string
-  service: RolesRule
-  currentApplication?: CurrentApplication
-  returnUrl: ReturnUrl
-}
+export type StaffRole = 'admin' | 'worker'
```
Comment (MargretFinnboga) [inline libs/financial-aid/shared/src/lib/types.ts:3]: worker, maybe bit strange maybe moderator or employee?

File: libs/financial-aid/shared/src/lib/interfaces.ts [hunk 3]
```diff
@@ -0,0 +1,168 @@
...
+  id: string
+  created: string
+  applicationId: string
+  eventType: ApplicationEventType
+  comment?: string
+}
+
+export interface Municipality {
+  id: string
+  // created: string
```
Comment (MargretFinnboga) [inline libs/financial-aid/shared/src/lib/interfaces.ts:81]: Take it out it out maybe, unused?

File: apps/financial-aid/backend/src/app/modules/staff/staff.service.ts [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+      municipalityId: 'hfj',
```
Comment (MargretFinnboga) [inline apps/financial-aid/backend/src/app/modules/staff/staff.service.ts:20]: Add phonenumber here ? 👀

File: apps/financial-aid/backend/src/app/modules/staff/staff.service.ts (hunk 1 above)
Comment (MargretFinnboga) [reply apps/financial-aid/backend/src/app/modules/staff/staff.service.ts:19]: Lets keep it like this for now, good start

File: apps/financial-aid/backend/src/app/modules/staff/staff.service.ts [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+import type { Staff } from '@island.is/financial-aid/shared/lib'
+
+@Injectable()
+export class StaffService {
+  constructor(
+    @InjectModel(StaffModel)
+    private readonly staffModel: typeof StaffModel,
+  ) {}
+
+  findById(id: string): Promise<Staff> {
```
Comment (MargretFinnboga) [inline apps/financial-aid/backend/src/app/modules/staff/staff.service.ts:15]: findby national Id might be more practical :)))

**Review by MargretFinnboga — COMMENTED** [review]
🎾  is in your court

File: apps/financial-aid/backend/src/app/modules/staff/models/staff.model.ts [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+
+  @ApiProperty()
+  municipalityId: string
+
+  @ApiProperty()
+  role: StaffRole
+
+  @ApiProperty()
+  active: boolean
+}
```
Comment (MargretFinnboga) [inline apps/financial-aid/backend/src/app/modules/staff/models/staff.model.ts:32]: okay soo minor comment, but missing phonenumber here

**Review by MargretFinnboga — APPROVED** [review]
Love the refactoring 
🏅

**Pull request merged**